.github/workflows/gradle.yml

```diff
@@ -77,5 +77,5 @@ jobs:
             uses: actions/upload-artifact@v3
             with:
                 name: plugin-json
-                path: build/reports/github-dependency-report/github-dependency-manifest.json
+                path: build/reports/github/github-dependency-snapshot.json
```

Comment:
Mind adding a rationale to the commit message why the new name makes more sense than the old name?

Reply:
The [GitHub Dependency Submission docs](https://docs.github.com/en/rest/dependency-graph/dependency-submission?apiVersion=2022-11-28) refer to the submitted data as a "snapshot".

> You can submit dependencies in the form of a snapshot. A snapshot is a set of dependencies associated with a commit SHA and other metadata, that reflects the current state of your repository for a commit.

Each snapshot may contain a number of `manifest` entries ([see schema here](https://docs.github.com/en/rest/dependency-graph/dependency-submission?apiVersion=2022-11-28#create-a-snapshot-of-dependencies-for-a-repository)).

As for changing the directory name, I was trying to avoid redundancy. But perhaps something related some `github-dependency-graph-plugin` would be more informative, given that's the source of the report.